### ISSUE TITLE
Find out why we get empty cells in column Speaker.Figure_id

### ISSUE THREAD

**Issue by nilsreiter** [quadrama/DramaNLP#78]
Related to https://github.com/quadrama/DramaAnalysis/issues/157.

**Comment by pagelj** [comment]
This happens when the Speaker IDs in the text are not identical with any ID in `<listPerson>`

For example, in Faust II (`11d11.0`), there is a choir in the beginning:
```{xml}
 <sp who="#chor">
    <speaker>CHOR</speaker>
        <stage>
            <hi>einzeln, zu zweien und vielen, abwechselnd und gesammelt.</hi>
         </stage>
         <lg>
            <l>Wenn sich lau die Lüfte füllen</l>
            <l>Um den grünumschränkten Plan,</l>
            <l>Süße Düfte, Nebelhüllen</l>
             <l>Senkt die Dämmerung heran.</l>
          </lg>

```

which is labeled as
```{xml}
 <person xml:id="chor_anmutige_gegend">
     <persName>CHOR (ANMUTIGE GEGEND)</persName>
 </person>
```
in `<listPerson>`

**Comment by pagelj** [comment]
After checking some more examples, this is primarily a problem with group characters such as `alle` etc.

**Comment by pagelj** [comment]
This got fixed in the current GDC version for the example above:

```{xml}
<sp who="#chor_anmutige_gegend">
   <speaker>CHOR</speaker>
    <stage>einzeln, zu zweien und vielen, abwechselnd und gesammelt.</stage>
```

**Comment by pagelj** [comment]
So upgrading the `qd` corpus to the current `gdc` version should fix the problem

**Comment by pagelj** [comment]
Also the GerDraCor reader needs to capture `<personGrp>` tags

**Comment by pagelj** [comment]
Fixed in 2b9f6239b0037af83019735e9beae5c6057aea35
